### PR TITLE
Create 2019-10-02-HackerEunji.md

### PR DESCRIPTION
**Pull request by HackerEunji**

### CODE REVIEW

**Comment by wanseob** [comment]
@HackerEunji Please change the file name from 09-02 to 10-02 :)

**Comment by wanseob** [comment]
@HackerEunji Would you please check the file name?

**Comment by HackerEunji** [comment]
I changed it! Can you check this out? Thank You for the comment:)

**Comment by wanseob** [comment]
@HackerEunji Thanks for changing the PR title but I meant the file name :) Would you please check it once again?

**Comment by HackerEunji** [comment]
Oh, I checked it and edited it agian. Now, am I correct? Sorry for the inconvenience and please let me know other mistakes I made! Thanks a lot!

File: _posts/2019-09-02-eunji.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 layout: contribution
 title: First attendance on October!
 category: meetup
-date:   2019-10-02 00:19:50 +0900
+date:   2019-09-02 00:19:50 +0900
```
Comment (wanseob) [inline _posts/2019-09-02-eunji.md]: Original one, 10-02, was right. What you've to change is the file name. Don't worry you can change it.

**Review by wanseob — CHANGES_REQUESTED** [review]

File: _posts/2019-09-02-eunji.md (hunk 1 above)
Comment (HackerEunji) [reply _posts/2019-09-02-eunji.md]: No way!! ;( I think I was confused. Is the file Name means " 2019-10-02-HackerEunji.md"?

**Comment by wanseob** [comment]
Goood  :+1:

**Pull request merged**